cmd/kubemark/hollow-node.go

```diff
@@ -161,6 +166,7 @@ func run(config *hollowNodeConfig) {
 
 	if config.Morph == "kubelet" {
 		f, c := kubemark.GetHollowKubeletConfig(config.NodeName, config.KubeletPort, config.KubeletReadOnlyPort, maxPods, podsPerCore)
+		f.NodeLabels = config.NodeLabels
```

Comment:
Nit: I think I would prefer to add additional argument to GetHollowKubeletConfig and set this there instead of adding custom logic here.

Another option is to pass entire config to the GetHollowKubeletConfig instead of config.NodeName, config.KubeletPort and others.